pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -401,7 +409,16 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 		// the processing logic needs to be added to the return value result.
 		if predicate.podTopologySpreadEnable {
 			_, status := podTopologySpreadFilter.PreFilter(context.TODO(), state, task.Pod)
-			if !status.IsSuccess() {
+			if status.IsSkip() {
```

Comment:
Abstract this logic to a function because many plugins do the same thing.

Reply:
done